fuzz-tests/Cargo.toml

```diff
@@ -9,7 +9,7 @@ cargo-fuzz = true
 
 [dependencies]
 libfuzzer-sys = { version = "0.4", optional = true }
-afl = { version = "0.12.16", features = ["reset_lazy_static"], optional = true }
+afl = { version = "0.12.16", optional = true }
```

Comment:
You can also remove the `[patch.crates-io]` section. It is no longer needed.

Reply:
Sounds good, I've removed that.